jep/200/README.adoc

```diff
@@ -276,7 +280,85 @@ For similar reasons, certain tests written in Groovy rather than Java prevent no
 
 == Reference Implementation
 
-link:https://github.com/jenkinsci/jenkins/pull/3120/files[Jenkins PR 3120] contains the bulk of the change and links to related PRs.
+* link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120] contains the bulk of the change and links to related PRs.
+* link:https://github.com/jenkinsci/remoting/pull/208[Remoting PR #208] introduces the new API required to deliver the change.
+
+== Rollout plan
+
+=== Phase 1. Remoting release
+
+Target date: Dec 29, 2017
+
+JEP-200 requires extra API in Remoting, so link:https://github.com/jenkinsci/remoting/pull/208[pull request #208]
+needs to be delivered in advance.
+https://github.com/oleg-nenashev[@oleg-nenashev] will release the following components once the rollout plan is confirmed with the JEP sponsor:
+
+* Remoting
+* link:https://github.com/jenkinsci/docker-slave[docker-slave] (including the `latest` tag)
+* link:https://github.com/jenkinsci/docker-jnlp-slave[docker-jnlp-slave] images (including the `latest` tag)
+* link:https://github.com/jenkinsci/swarm-plugin/[Swarm Plugin Client]
```

Comment:
All irrelevant for purposes of this JEP: only the `remoting.jar` loaded inside `jenkins.war` needs to be updated for the change to take effect. Agents can run whatever version.

Reply:
To be clear, obviously you can and should bump up various images and other consumers of Remoting after a release, but that has nothing to do with this JEP and need not be discussed here.

Reply:
I consider it as a part of the Remoting rollout, which should get us some soak testing of the released lib. Note that the Remoting release may include other changes, and we do not want to revert JEP due to the regression in other components. That's why I want to be explicit that all packaging components get updated timely.